product_docs/docs/pgd/5.6/nodes/subscriber_only/creating-so.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Creating Subscriber-only groups and nodes
+description: 'Creating Subscriber-only groups and nodes in EDB Postgres Distributed'
+---
+
+The process of creating a Subscriber-only node or nodes starts with creating a Subscriber-only group to contain the node or nodes.
+Perform this  step on an existing fully joined node in the PGD cluster.
+
+## Creating a Subscriber-only group manually
+
+To create a Subscriber-only group, you must specify the `node_group_type` as `subscriber-only` when creating the group. For example, here we are logged into the node "node-one" running on "host-one". It's a member of it's own data group and as for all nodes, a member of the top-level group, here called `topgroup`. Log into this node directly to create a new Subscriber-only group named `sogroup` with the following SQL command:
+
+```sql
+select bdr.create_node_group('sogroup', 'topgroup', false, 'subscriber-only');
+```
+
+or more explicitly with parameter names:
+
+```sql
+select bdr.create_node_group(node_group_name:='sogroup',
+        parent_group_name:='topgroup',
+        join_node_group:=false,
+        node_group_type:='subscriber-only');
+```
+
+This creates a Subscriber-only group named `sogroup` which is a child of the `topgroup` group. The false parameter for `join_node_group` indicates that the node executing this command shouldn't join to the newly created group. Automatically joining the group is the default behavior, which in this case needs to be supressed.
+
+## Adding a node to a new Subscriber-only group manually
+
+You can now initialise a new data node and then add it to the Subscriber-only group. Create a data node and configure the bdr extension on it as you would for any other data node.
```

Comment:
> and configure the bdr extension

Maybe just link here the setup page for a node.

Reply:
Oddly we lack a page on creating a data node outside the manual deployment guide. A hang over from the TPA-only era. 

```suggestion
You can now initialise a new data node and then add it to the Subscriber-only group. Create a data node and configure the bdr extension on it as you would for any other data node; For example, if you deployed manually, see the [manual install guide]/pgd/latest/deploy-config/deploy-manual/deploying/04-installing-software/).
```

This does suggest a "bare" how to create a data node guide is needed.